runtime/mainnet/src/xcm_config.rs

```diff
@@ -95,8 +96,11 @@ pub type BridgedLocalAssetTransactor = FungibleAdapter<
 	(),
 >;
 
+pub type DotBurnerTransactor = BurnerAdapter<IsConcrete<RelayLocation>>;
```

Comment:
I've perhaps make the name more generic. DOT is in Polkadot, PAS in Paseo. What about `RelayTokenBurnerTransactor`?